openssl/src/ssl/mod.rs

```diff
@@ -1880,6 +1890,16 @@ impl SslContextRef {
         let mode = unsafe { ffi::SSL_CTX_get_verify_mode(self.as_ptr()) };
         SslVerifyMode::from_bits(mode).expect("SSL_CTX_get_verify_mode returned invalid mode")
     }
+
+    /// Gets the number of TLS 1.3 session tickets that will be sent to a client after a full
+    /// handshake.
+    ///
+    /// Requires OpenSSL 1.1.1 or newer.
+    #[corresponds(SSL_CTX_get_num_tickets)]
+    #[cfg(ossl111)]
+    pub fn get_num_tickets(&self) -> usize {
```

Comment:
Getters in Rust don't usually use the `get_` prefix, so this would just be `pub fn num_tickets`.